format/FormatNumber.js

```diff
@@ -150,15 +150,20 @@ export function fmtQuantity(v, opts = {}) {
     saveOriginal(v, opts);
     if (isInvalidInput(v)) return fmtNumber(v, opts);
 
-    const lessThanM = Math.abs(v) < MILLION;
+    const lessThanM = Math.abs(v) < MILLION,
+        greaterThanB = Math.abs(v) >= BILLION;
```

Comment:
for symmetry can we define the second clause as lessThanB and then have three cases
if (lessThanM) return fmt()
if (lessThanB) return fmtMillions()
return fmtBillons()

Reply:
also need to updates the jsdoc to reflect the new behavior